src/core/jit/exec.rs

```diff
@@ -70,6 +71,28 @@ where
     async fn init(&mut self) {
         join_all(self.plan.as_children().iter().map(|field| async {
             let ctx = Context::new(&self.request);
+            let mut arg_map = indexmap::IndexMap::new();
+            for arg in field.args.iter() {
```

Comment:
Field will only have arg names but not the actual args. Actual arg information is in the request.

Reply:
If I understand correctly, the way we are parsing the query, we will find the value of arguments in the `Arg` type when query is passed as `query {user(id: 1) {id}}` and when it is passed as `query user($id: Int!) {user(id: $id) { id }}` then we will find the value in variables